Vehicles/Electric_transport/Electric_transport.py

```diff
@@ -1,7 +1,7 @@
-from python_lab9.Vehicles.Vehicles import Vehicles
+from Vehicles.Vehicles import Vehicles
 
 
-class Electric_transport(Vehicles):
-    def __init__(self, vehicles_type, vehicles_capacity_of_people, vehicles_price, electricity_needed_per_hour):
+class ElectricTransport(Vehicles):
+    def __init__(self, vehicles_type, vehicles_capacity_of_people, vehicles_price, electricity_needed_per_hour_in_watts):
```

Comment:
Please add [type annotations](https://docs.python.org/3/library/typing.html) to your classes